modules/tensor_mechanics/include/materials/ComputeCappedWeakPlaneStress.h

```diff
@@ -7,7 +7,7 @@
 #ifndef COMPUTECAPPEDWEAKPLANESTRESS_H
```

Comment:
Sorry, but I think we should rename this to `CappedWeakPlaneStressUpdate` to be consistent with the other models.